packages/skill-lesson/trpc/routers/progress.ts

```diff
@@ -118,7 +118,7 @@ export const progressRouter = router({
           })
 
           const moduleProgress = await getModuleProgress({
-            moduleId: progress.moduleId ?? '',
+            moduleId: input.moduleId ?? '',
             userId: token.sub,
```

Comment:
using the input that is passed in will be more resilient so that if for some reason progress is undefined, we can still retrieve module progress